constitution.md

```diff
@@ -0,0 +1,196 @@
+# Constitution of the Unix Users Group
+
+October 25, 1999  
+Authored by David Wilburn,  
+Amended, May 1, 2003.  
+Amended, April 1, 2019.  
+
+## Article I - Name of Organization
+
+This organization shall be called the James Madison University Unix Users
+Group. The Unix Users Group is also referred to as the UUG.
+
+## Article II - Purpose of Organization
+
+This organization is devoted to users of the Unix computer operating system.
+Its mission:
+- To promote Unix awareness and knowledge, and
+- To provide a means of communication between persons having interest in the
+Unix operating system.
+
+## Article III - Membership
```

Comment:
How do we define a member? Do we want GPA requirements? Are we open to the community outside JMU? What right and privileges come with membership vs just having open meetings?

Reply:
I think it's also important to clarify at least in our own minds (if not in the document) how subscription to GEEK-L relates (or does not relate) to membership.

I think we've tossed around the idea in the past of having required membership dues each semester (maybe $5 or $10) to help build a stronger sense of community investment and distinguish those who simply attend from those who are contributing members. The dues could go towards equipment for demos or in reimbursing external speakers for travel expenses.

Also, perhaps we could waive (or reduce) membership dues for members who commit to volunteering to lead a tutorial that semester as a further incentive to contributing.